src/scales/colorScale.ts

```diff
@@ -60,7 +60,7 @@ export module Scale {
 
     private static _getPlottableColors(): string[] {
       var plottableDefaultColors: string[] = [];
-      var colorTester = d3.select("body").append("div");
+      var colorTester = d3.select("body").append("plottable");
```

Comment:
What we were doing before was a hack anyway. Elements don't have to exist in order to have CSS classes attached to them